packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -168,19 +170,21 @@ export function setValueForProperty(
     const {type} = propertyInfo;
     let attributeValue;
     if (type === BOOLEAN || (type === OVERLOADED_BOOLEAN && value === true)) {
+      // If attribute type is boolean, we know for sure it won't be an execution sink
+      // and we won't require Trusted Type here.
       attributeValue = '';
     } else {
       // `setAttribute` with objects becomes only `[object]` in IE8/9,
       // ('' + value) makes it output the correct toString()-value.
```

Comment:
Outdated comment

Reply:
Mhh, isn't it still valid? I mean that is the reason why we use `toStringOrTrustedType`.

Reply:
Yeah but I mean since literally `('' + value)`  is not present, people might think this whole thing is irrelevant. But it's still relevant.

I think it's fine though.